citesphere/src/main/webapp/WEB-INF/views/auth/group/item.html

```diff
@@ -111,7 +111,11 @@
 	            success: function(data) {
 	                document.getElementById('uploadFileInput').value = '';
 	                $("#uploadFileModal").modal('hide');
-	            }
+	            },
+		        error: function(data) {
+		        	$("#uploadFileModal").modal('hide');
+		            showNotification(" Error while attempting to upload files!", "danger", 10000);
```

Comment:
indentation but also, the error should not fade out, otherwise if the user leaves to get a coffee after clicking upload or gets distracted and then come back they won't see the error message.